.github/workflows.disabled/plagiarism-checker.yml

```diff
@@ -94,6 +125,6 @@ jobs:
             issue_number: context.payload.pull_request.number,
             owner: context.repo.owner,
             repo: context.repo.repo,
-            body: "Here's a screenshot of the article (/engineering-education/${{ matrix.article_slug }}/):\n\n![screenshot](${{ steps.upload-screenshot.outputs.img_url }})"
+            body: "Here's your plagiarism check report (/engineering-education/${{ matrix.article_slug }}/):\n\n![plagiarism-check](${{ steps.output_report.outputs.${{ id }}.pdf }})"
```

Comment:
If I've got this right, you're going to store the report PDF as an artifact on the workflow, and then link to it from the PR?

That's pretty cool!